i18n/chs/out/commands.i18n.json

```diff
@@ -1,5 +1,6 @@
 {
   "autolispext.help.commands.openWebHelp": "无法加载 webHelpAbstraction.json 文件",
   "autolispext.commands.addFoldingRegion": "无法插入代码段",
-  "autolispext.help.commands.generateDocumentation": "找不到有效的 defun 名称"
+  "autolispext.help.commands.generateDocumentation": "找不到有效的 defun 名称",
```

Comment:
We don't need to do it on our own. Before publishing a new release, we'll send English resource files to the globalization team, and they'll provide us with all the non-English resource files.